types/sinon-chrome/index.d.ts

```diff
@@ -353,6 +356,23 @@ declare namespace SinonChrome.permissions {
     export var request: SinonChromeStub;
 }
 
+declare namespace SinonChrome.plugins {
+    export interface Plugin {}
+    export interface Translation {
```

Comment:
Maybe follow the browser spec:

```javascript
export interface Translation {
  message: string;
  description?: string;
  placeholders?: {
    [key: string]: {
      content: string,
      example?: string,
    }
  };
}
```

Reply:
Thanks for the feedback.
I follow the browser spec.